### PR TITLE
Feature/custom locations

### PR DESCRIPTION
**Pull request by kolbii**
I've been using this tool for making requests between the projects and services, it's a great tool for that. But in that case, if a project has 2+ location path it gets difficult to use and understand.
Please take a look at this PR. I've added a new pane to Proxy host dialog as "Custom locations" where you'd be able to manaeg 2+ location blocks

[screenshot1](https://imgur.com/3ffSp5y)
[screenshot2](https://i.imgur.com/gsFUaAc.png)

### CODE REVIEW

**Comment by jc21** [comment]
Ignore the CI errors for now, gotta get my CI pipelines working with PRs.

**Comment by jc21** [comment]
Docker Image for build 5 is available on [DockerHub](https://cloud.docker.com/repository/docker/jc21/nginx-proxy-manager) as `jc21/nginx-proxy-manager:github-pr-74`

**Comment by jc21** [comment]
Docker Image for build 6 is available on [DockerHub](https://cloud.docker.com/repository/docker/jc21/nginx-proxy-manager) as `jc21/nginx-proxy-manager:github-pr-74`

**Comment by jc21** [comment]
Docker Image for build 7 is available on [DockerHub](https://cloud.docker.com/repository/docker/jc21/nginx-proxy-manager) as `jc21/nginx-proxy-manager:github-pr-74`

**Comment by jc21** [comment]
I've been a bit busy I'll try to review this properly this week

**Comment by jc21** [comment]
Docker Image for build 8 is available on [DockerHub](https://cloud.docker.com/repository/docker/jc21/nginx-proxy-manager) as `jc21/nginx-proxy-manager:github-pr-74`

File: src/backend/templates/_location.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+  location {{ path }} {
+    proxy_set_header Host $host;
+    proxy_set_header X-Forwarded-Scheme $scheme;
+    proxy_set_header X-Forwarded-Proto  $scheme;
+    proxy_set_header X-Forwarded-For    $remote_addr;
+    proxy_pass       {{ forward_scheme }}://{{ forward_host }}:{{ forward_port: }};
```
Comment (jc21) [inline src/backend/templates/_location.conf]: Looks like a typo, an extra colon in `forward_port:`

File: src/frontend/js/app/nginx/proxy/location-item.ejs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                                </span>
+                                <input type="text" name="path" class="form-control model" value="<%- path %>" placeholder="<%- i18n('locations', 'path') %>" required>
+                            </div>
+                        </div>
+                        <div class="col-auto">
+                            <div class="selectgroup">
+                                <label class="selectgroup-item">
+                                    <input type="checkbox" class="selectgroup-input">
+                                    <span class="selectgroup-button">
+                                        <i class="fe fe-eye"></i>
```
Comment (jc21) [inline src/frontend/js/app/nginx/proxy/location-item.ejs]: Small improvement, the icon should be `fe-settings` to be consistent with the advanced settings for the main host

**Comment by jc21** [comment]
Docker Image for build 9 is available on [DockerHub](https://cloud.docker.com/repository/docker/jc21/nginx-proxy-manager) as `jc21/nginx-proxy-manager:github-pr-74`

**Comment by kolbii** [comment]
Thanks for the feedback. You can find the fixes in the last commit

**Comment by jc21** [comment]
Docker Image for build 10 is available on [DockerHub](https://cloud.docker.com/repository/docker/jc21/nginx-proxy-manager) as `jc21/nginx-proxy-manager:github-pr-74`

**Pull request merged**